lib/fontv/utilities.py

```diff
@@ -44,8 +44,18 @@ def get_git_root_path(filepath):
 
     # search up to five directories above for the git repo root
     for _ in range(6):
-        if dir_exists(os.path.join(gitroot_path, ".git")):
+        dot_git = os.path.join(gitroot_path, ".git")
```

Comment:
Before you assume that `.git` exists at the root of a repository, first check `$GIT_DIR` from the environment. The defaults are obviously in common use but exceptions are not so uncommon that it is smart to just ignore them. Unfortunately anywhere you are running `font-v` might not actually by inside a shell environment where the Git dir has been setup for the benefit of other tools, but it's better than nothing.

Reply:
Yes, just something I happened to notice when reviewing this.